src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
@stephentoub @danmosemsft @karelz  - Any opinions on how to use ActiveIssue(n)....should 'n' be a number or a URL string?  It seems to me that it should be a number especially since it is more than likely to be an issue in the same repo as the tests.  I find the URL cumbersome.

It would be nice to get a consistent pattern to use in the code.